### ISSUE TITLE
downloading a file problem

### ISSUE THREAD

**Issue by andrei4002** [Files-com/files-sdk-ruby#10]
Hello, I'm trying to download a file using the following code:

```
file = Files::Dir.list_for("some/path/to/a/folder/with/jpeg/files").first
file.download_file(File.join(Rails.root, "tmp", "test.jpeg"))
```

Something does get downloaded but it's definitely not the jpeg file. the downloaded file also doesn't seem to have the same file size, looks to be actually double in size - this happens to any file i try to download like this

any pointers if downloading should be done differently or if there's an issue somewhere?

**Comment by zeisler** [comment]
Hello @andrei4002,

I'm looking into your issue to see if I reproduce this and will come back with a suggestions and/or fixes.

**Comment by zeisler** [comment]
@andrei4002 I am not able to reproduce your issue. Are you able to download this file from the Web UI correctly?

**Comment by zeisler** [comment]
Also are you using regular storage or is this stored on a Remote Server Mount?

**Comment by andrei4002** [comment]
i can download the files from the web ui correctly

not sure about storage, i'm using whatever's default on files.com, and calling the functions on my local machine (mac os 11, ruby 2.6.5, Rails 6.0.2.1)

**Comment by zeisler** [comment]
What's your site name and I'll see if I can see anything in the logs?

**Comment by bombino** [comment]
To clarify, please email support@files.com with that information and reference this thread.  Don't want you to have to provide that publicly.

Best,
Kevin Bombino
Files.com

**Comment by zeisler** [comment]
This error was only happening in the Rails context and didn't happen while running in plain Ruby. This has been fixed as of [1.0.131](https://rubygems.org/gems/files.com/versions/1.0.131). Let me know if you have any additional issues.
